jormungandr/src/network/subscription.rs

```diff
@@ -57,7 +59,8 @@ where
     tokio::spawn(
         inbound
             .for_each(move |fragment| {
-                // TODO: implement
+                let msg = TransactionMsg::SendTransaction(FragmentOrigin::Network, vec![fragment]);
+                transaction_box.try_send(msg).unwrap();
```

Comment:
I did not get to this place last time. Would it benefit from the same treatment?

Reply:
This is coming from the outside, so if we run out of MPSC buffer, we probably should just drop it instead of queuing on Tokio.

I'll change it so it ignores full queue error, but returns error in other cases.

Reply:
No, the correct thing is to implement back pressure that will pause the inbound stream. I've looked at the similar piece of code that processes incoming blocks and it's wrong in the same way, except that blocks are large and will probably be processed faster than they are received.

Reply:
I can change both places in a follow-up PR.

Reply:
Thanks! So this comment is resolved?

Reply:
Yes.